examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -0,0 +1,294 @@
+#
+# Running from command line:
+#
+# Server machine's IP address, port number, and multiple comma separated resource names can be passed as separate command line arguments.
+#   > python graph-measurement.py <server_address> <port_number> <resource_name1,resource_name2>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763", with "SimulatedScope1" and "SimulatedScope2" as the resource names
+
+
+import grpc
+import niscope_pb2 as niscope_types
+import nitclk_pb2 as nitclk_types
+import niscope_pb2_grpc as grpc_scope
+import nitclk_pb2_grpc as grpc_tclk
+import matplotlib.pyplot as plt
+import sys
+
+server_address = "localhost"
+server_port = "31763"
+resources = []
+
+# Resource name and options for a simulated 5122 client. Change them according to the NI-SCOPE model.
+resource1 = "SimulatedScope1"
+resource2 = "SimulatedScope2"
+options = "Simulate=1, DriverSetup=Model:5122; BoardType:PCI;"
+resources.extend([resource1, resource2])
+
+# Parameters
+channels = "0"
+previous_max_input_frequency = -10
+previous_min_sample_rate = -10
+previous_min_record_length = -10
+previous_trigger_type = -10
+trigger_type = -10
+max_input_frequency = 0.0
+min_sample_rate = 1000000
+num_records = 1
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    server_address = sys.argv[1]
+if len(sys.argv) >= 3:
+    server_port = sys.argv[2]
+if len(sys.argv) >= 4:
+    resources = sys.argv[3]
+    resources = resources.split(',')
+    options = ""
+
+# Create the communication channel for the remote host (in this case we are connecting to a local server)
+# and create a connection to the NI-SCOPE service and NI-TCLK service
+channel = grpc.insecure_channel(f"{server_address}:{server_port}")
+scope_service = grpc_scope.NiScopeStub(channel)
+tclk_service = grpc_tclk.NiTClkStub(channel)
+sessions = []
+
+# Checks for errors. If any, throws an exception to stop the execution.
+any_error = False
+def CheckForError (service, vi, status) :
+    global any_error
+    if(status != 0 and not any_error):
+        any_error = True
+        ThrowOnError (service, vi, status)
+
+def ThrowOnError (service, vi, error_code):
+     if service == scope_service:
+            error_message_request = niscope_types.GetErrorMessageRequest(
+                vi = vi,
+                error_code = error_code
+            )
+            error_message_response = scope_service.GetErrorMessage(error_message_request)
+     else:
+            error_message_request = nitclk_types.GetExtendedErrorInfoRequest()
+            error_message_response = tclk_service.GetExtendedErrorInfo(error_message_request)           
+     raise Exception (error_message_response.error_string)
+
+try :
+    #Initialize the session
+    i=0
+    for resource in resources:
+        init_result = scope_service.InitWithOptions(niscope_types.InitWithOptionsRequest(
+            session_name = "demo" + str(i),
```

Comment:
Currently session names are going to something like:
demo0, demo1, demo2 etc

Instead it would be better to have something like:
session1, session2, session3 etc

for that, in line 100, we can make it i=1 and change "demo" to "session"